.travis.yml

```diff
@@ -1,22 +1,20 @@
 language: c
 install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
 script: bash -ex .travis-docker.sh
-sudo: required
 services:
   - docker
 sudo: false
 env:
  global:
-   - PRE_INSTALL_HOOK="cd /home/opam/opam-repository && git pull origin master && opam update -u -y"
```

Comment:
I think this is still useful -- why remove it?

Reply:
because it is already done in [.travis-docker.sh](https://github.com/ocaml/ocaml-ci-scripts/blob/4466db9737c1e615d4aab631bb32b3505ea68ce6/.travis-docker.sh#L78)